README.md

```diff
@@ -1,9 +1,8 @@
 # 🤔 Introduction :
-Ternoa is a Decentralised, Open source, NFT-centric Layer 1 blockchain that is multi-chain by design and aims to provide a technical stack to build scalable and secure NFTs with native support for advanced features.
 
 For Builders By Builders :
 
-NFTs native to our chain can be deployed using High-level programming languages and does not require smart contract functionality.
+NFTs native to our chain can be minted using High-level programming languages and does not require smart contract functionality.
 
 Privacy, A Fundamental Right :
```

Comment:
This one can be removed.

Reply:
Also 'Sustainability is Key'

Reply:
Just to be sure, Do I remove the description of the section "For Builders By Builders"?

And keep the privacy part

Reply:
# 🤔 Introduction :

Ternoa is a Decentralised, Open source, NFT-centric Layer 1 blockchain that is multi-chain by design and aims to provide a technical stack to build scalable and secure NFTs with native support for advanced features.

##### For Builders By Builders :

With native support for Secret NFTs, Delegating and Lending, Transaction Batching and much more, you might want to try.

##### Native support for Advanced Features :

With native support for Secret NFTs, Delegating and Lending, Transaction Batching and much more, you might want to give it a try.

##### Ecosystem 
Our ecosystem of NFT-based Dapps keeps growing day after day. Our SDK relies upon the most popular high-level languages, allowing us to tap into the world’s largest pool of existing developers, thereby minimizing the transition period.

### Contribution Guidelines :

As Ternoa-js is an open-source SDK, feel free to interact with the tools and libraries, log issues, create pull requests or leave feedback. We welcome and greatly appreciate your contribution.

If you’re interested in contributing to the Ternoa SDK, we recommend you inspect our [contribution guidelines](https://github.com/capsule-corp-ternoa/ternoa-js/blob/main/CONTRIBUTING.md).

If you want to learn how to use the Ternoa SDK, the [test-dapp](https://github.com/capsule-corp-ternoa/ternoa-js-test-dapp) is the perfect place to familiarize yourself with our architecture and contribute to its development.

### Error Reporting :
If you encounter any errors along the way, technical or otherwise. Let us know and we'll deal with it swiftly. 
It'll help us further improve the overall experience for our users.

* Open a discussion of type `General` in the [discussions section](https://github.com/capsule-corp-ternoa/ternoa-js/discussions) if you encounter any unexpected behaviour.
* Open a Bug report using the [bug template](https://github.com/capsule-corp-ternoa/ternoa-js/issues/new/choose) if the bug persists.
* If you can, suggest a fix in a pull request to resolve that issue.

Make sure to document the error properly, keeping in mind that the better you describe it, the easier it is to deal with.

Reply:
This is how I see the introduction section 👆

Reply:
For Builders By Builders :
With native support for Secret NFTs, Delegating and Lending, Transaction Batching and much more, you might want to try.

Native support for Advanced Features :
With native support for Secret NFTs, Delegating and Lending, Transaction Batching and much more, you might want to give it a try.


This section was doubled so I just Kept One of em

Reply:
Ternoa is a Decentralised, Open source, NFT-centric Layer 1 blockchain that is multi-chain by design and aims to provide a technical stack to build scalable and secure NFTs with native support for advanced features.

For Builders By Builders :
With native support for Advanced features such as Secret NFTs, Delegating and Lending, Transaction Batching and much more, you might want to try.

Ecosystem :
Our ecosystem of NFT-based Dapps keeps growing day after day. Our SDK relies upon the most popular high-level languages, allowing us to tap into the world’s largest pool of existing developers, thereby minimizing the transition period.

Reply:
Ternoa is a Decentralised, Open source, NFT-centric Layer 1 blockchain that is multi-chain by design and aims to provide a technical stack to build scalable and secure NFTs with native support for advanced features.

For Builders By Builders :

NFTs native to our chain can be minted using High-level programming languages and doesn't require smart contract functionality.

Native support for Advanced Features :

With native support for Secret NFTs, Delegating and Lending, Transaction Batching and much more, you might want to give it a try.

Ecosystem

Our ecosystem of NFT-based Dapps keeps growing day after day. Our SDK relies upon the most popular high-level languages, allowing us to tap into the world’s largest pool of existing developers, thereby minimizing the transition period.

Reply:
Done